src/examples/app_v2.py

```
@@ -0,0 +1,57 @@
+__license__ = "Apache License, Version 2.0"
+""" The license for the module """
+
+import base
+
+import appier
+
+
+class MBAppV2(appier.APIApp):
+
+    def __init__(self, *args, **kwargs):
+        appier.APIApp.__init__(self, name="mb_v2", *args, **kwargs)
```

Ruff (0.8.0)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)